### PR TITLE
1、Distinguish between normal and error output logs, which is convenient for packaging scripts to analyze the reasons

### PR DESCRIPTION
**Pull request by lingsg**
2、Compatible with ilmerge's allowDup parameter handling (without the class name after it)

### CODE REVIEW

File: ILRepack.IntegrationTests/Scenarios/DotNet462Application/App.config [hunk 1]
```diff
@@ -1,13 +1,13 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <configuration>
     <startup> 
-        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.2" />
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1"/>
```
Comment (lextm) [inline ILRepack.IntegrationTests/Scenarios/DotNet462Application/App.config:4]: It makes no sense to change target .NET Framework version here. .NET Framework 4.6.2 is currently the minimal 4.x version to be supported by Microsoft.

File: ILRepack.IntegrationTests/Scenarios/DotNet462Application/App.config [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
     </startup>
   <runtime>
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
-        <assemblyIdentity name="System.Runtime" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-4.1.1.0" newVersion="4.1.1.0" />
```
Comment (lextm) [inline ILRepack.IntegrationTests/Scenarios/DotNet462Application/App.config:10]: The change on tag ending is also annoying. What editor are you using? A space before `/>` is the Microsoft common style.

File: ILRepack/ILRepack.csproj [hunk 3]
```diff
@@ -51,7 +51,7 @@
   <ItemGroup>
     <PackageReference Include="BamlParser" Version="1.0.0" />
     <PackageReference Include="fasterflect" Version="2.1.3" />
-    <PackageReference Include="Mono.Posix" Version="4.0.0" />
+    <PackageReference Include="Mono.Posix-4.5" Version="4.5.0" />
```
Comment (lextm) [inline ILRepack/ILRepack.csproj:54]: Why switching to an out-of-maintenance package?

File: ILRepack/ILRepack.csproj [hunk 4]
```diff
@@ -4,7 +4,7 @@
     <ProjectGuid>{4A253A60-D998-4CA2-B9D5-46567A2FBF80}</ProjectGuid>
     <OutputType>Exe</OutputType>
     <RootNamespace>ILRepacking</RootNamespace>
-    <TargetFramework>net40</TargetFramework>
+    <TargetFramework>net45</TargetFramework>
```
Comment (lextm) [inline ILRepack/ILRepack.csproj:7]: This change is also questionable. Any reason to switch to 4.5 profile?

**Comment by KirillOsenkov** [comment]
I applied the Console.Error changes here:
https://github.com/BSimonSweet/il-repack/commit/ec6ea0570e171dc74dd94173391ad60ffce84160

**Comment by KirillOsenkov** [comment]
I applied the -allowdup modifier logic here:
https://github.com/gluck/il-repack/commit/2dbf68f53655b024d2802b1e1e8078c3190fa48b

The rest of the changes look accidental and are not useful.

Thanks!

**Pull request closed without merging**